### DISCUSSION TITLE
In Search of The Golden Soroban

### DISCUSSION THREAD

**Discussion by StellarPunkCaptain** [stellar/sorobanathon#154] [General]
Hello Everyone!, this is my submission for the 7 chapter art challenge. you will find a preview of each piece below and the link to the NFTs on Litemint. please enjoy my take on this story as The Stellar Punks go in search of the Legendary Golden Soroban! 

The Prologue
![Prologue](https://user-images.githubusercontent.com/119875876/207769649-ca016374-9c28-44c7-a803-19594affd1b8.png)
https://litemint.com/items/GBNU32XJGTZ55Y7J7SYMSR2VUBMXERBOYURZVGSAZZV46TGAHC2L4TEO/prologue

Chapter 1
![Chapter 1](https://user-images.githubusercontent.com/119875876/207769669-78aa286f-58ea-442e-9a4e-a7f43b890fce.png)
https://litemint.com/items/GCJTNPJZMNCZY2YWJLCRFFAZU7NTNSAX7VQLZRGNJ56BOGHS7CPIXR2H/chapter1

Chapter 2
![Chapter 2](https://user-images.githubusercontent.com/119875876/207769704-09629c6d-91ed-4c35-a74a-ca2696d3d14e.png)
https://litemint.com/items/GBF7FIA5X2GD6YNPMLO2KDWZUNXL5Q4TC4J4TSQCE66GVM4VAO3YVW44/chapter2

Chapter 3 
![Chapter 3](https://user-images.githubusercontent.com/119875876/207769715-d82c1167-3f00-47c5-8fe7-839db4552499.png)
https://litemint.com/items/GAJ6LLPOIPAWBROKF4FDW4NL3HG3OEHOSGDZEPDOY6AMJOMQBM3BG4SI/chapter3

Chapter 4
![Chapter 4](https://user-images.githubusercontent.com/119875876/207772936-68e38105-bfa4-4ebb-a4b5-7716e25328d9.png)
https://litemint.com/items/GBG7U34N6SZ65MSGWCW2M4SXIF5MHH5ASXVBQKX63HZ36MICS5SH7FK5/chapter4

Chapter 5
![Chapter 5](https://user-images.githubusercontent.com/119875876/207769753-20cdd9a2-130b-4aa2-9364-98203bb8e495.png)
https://litemint.com/items/GC3LVAN6WAJMVN7GAMKWHKK3JJGIR4I2227B7QKNYZK2SDAYTSJFZUVU/chapter5

Chapter 6
![chapter 6](https://user-images.githubusercontent.com/119875876/207769774-4bbc676a-bea4-48bd-a285-8cf4dbda1cf3.png)
https://litemint.com/items/GAHJKKDRDKPLJDTRE3WRJWDNJY3C2BN7ZWMUN53L2X2JVRTONH6S6DA2/chapter6

Thank You Everyone for this amazing experience!

**Comment by rawritude** [comment]
Woooooow fantastic job Andrew!!!!

**Reply by silence48** [reply]
nice work Andrew! I love them!

**Comment by StellarPunkCaptain** [comment]
Thanks Guys!

**Comment by RDCrypto** [comment]
Really cool. Andrew! Nice job!

**Reply by StellarPunkCaptain** [reply]
Thank you Rick!

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John

**Reply by StellarPunkCaptain** [reply]
Thank You so much!! filling out the form right now
